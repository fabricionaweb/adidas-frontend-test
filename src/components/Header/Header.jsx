import React from 'react';
import styles from './Header.module.css';

export function Header() {
  return (
    <header className={styles.wrapper}>
      <h1 className={styles.heading}>
        <p>Sign up form</p>
        Get more information
      </h1>
    </header>
  );
}

export default Header;
