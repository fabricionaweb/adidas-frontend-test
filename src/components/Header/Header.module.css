.wrapper {
  background-color: #E1E1E1;
  padding: 1.5rem 0 3rem;
}

.heading {
  text-transform: uppercase;
  font-weight: 800;
  font-size: 3rem;
  margin: 0 auto;
  padding: 0;
  max-width: 900px;
  width: 92%;
}

.heading p {
  margin: 0 0 0.5rem;
  font-size: 1rem;
  font-weight: 400;
  text-transform: none;
}
