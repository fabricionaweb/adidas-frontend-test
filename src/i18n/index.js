import i18n from 'i18next';
import { initReactI18next } from 'react-i18next';
import resources from './resources';

i18n
  .use(initReactI18next) // passes i18n down to react-i18next
  .init({
    resources,
    lng: 'en', // default language
    fallbackLng: 'en', // use en if detected lng is not available
    keySeparator: false, // we do not use keys in form messages.welcome
    debug: process.env.NODE_ENV.includes('dev'),
    interpolation: {
      escapeValue: false, // react already safes from xss
    },
  });

export default i18n;
