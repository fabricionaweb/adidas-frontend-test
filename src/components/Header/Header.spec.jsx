import React from 'react';
import { render } from '@testing-library/react';
import { Header } from './Header';
import '../../i18n';

describe('Header.jsx', () => {
  it('should render without crashing', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
