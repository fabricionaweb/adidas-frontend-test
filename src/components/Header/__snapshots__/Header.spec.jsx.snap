// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.jsx should render without crashing 1`] = `
<DocumentFragment>
  <header
    class="wrapper"
  >
    <h1
      class="heading"
    >
      <p>
        Sign up form
      </p>
      Get more information
    </h1>
  </header>
</DocumentFragment>
`;
